src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java

```diff
@@ -47,7 +47,7 @@ public class ConditionRelation extends UniqueIdAuditedObject {
     @IndexedEmbedded(includeDepth = 1)
     @IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
     @ManyToMany
-    @JsonView({View.FieldsAndLists.class})
+    @JsonView({View.FieldsOnly.class})
```

Comment:
I would guess this is causing the IT's to fail? Maybe we create a separate view for the specific operation.

Reply:
Ok, I created a new update method.

Reply:
Just override the current one we dont want two